docs/tutorials/model_to_instance.ipynb

```diff
@@ -0,0 +1,314 @@
+    "# 目的関数\n",
+    "problem += jm.sum(i, v[i] * x[i])\n",
+    "# 制約条件: ナップサックの耐荷重を超えない\n",
+    "problem += jm.Constraint(\"重量制限\", jm.sum(i, w[i] * x[i]) <= W)\n",
+    "problem"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step2. Interpreter オブジェクトにインスタンスデータを登録する\n",
+    "\n",
+    "Step1で定式化した数理モデルの係数に入力するデータ（インスタンスデータ）を用意します。これは `Interpreter` オブジェクトを使用して実現できます。\n",
+    "\n",
+    "`Interpreter` クラスのコンストラクタの引数に、以下のペアを持つ辞書を渡すことでインスタンスデータを登録できます：\n",
+    "\n",
+    "- キー：`Placeholder` オブジェクトの `name` プロパティに設定した文字列\n",
+    "- 値：入力するデータ"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "interpreter = jm.Interpreter({\n",
+    "    \"v\": [10, 13, 18, 31, 7, 15],  # アイテムの価値のデータ\n",
+    "    \"w\": [11, 15, 20, 35, 10, 33], # アイテムの重さのデータ\n",
+    "    \"W\": 47,                       # ナップサックの耐荷重のデータ\n",
+    "})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step3. Interpreter オブジェクトを使って数理モデルをインスタンスに変換する\n",
+    "\n",
+    "数理モデルをインスタンスデータに変換するには、Interpreter.eval_problem メソッドを使用します。インスタンスデータが登録された Interpreter オブジェクトの eval_problem メソッドに Problem オブジェクトを渡すと、その Problem オブジェクトが持つ Placeholder にデータを入力し、インスタンスに変換して ommx.v1.Instance オブジェクトとして返します："
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "instance = interpreter.eval_problem(problem)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## OMMX Adapterに入力して最適化問題を解く\n",
+    "\n",
+    "Step3で得られた `ommx.v1.Instance` オブジェクトをOMMX Adapterに入力することで、最適化計算を実行できます。\n",
+    "例えば、OMMX Adapterの一つである `ommx-python-mip-adapter` ([GitHub](https://github.com/Jij-Inc/ommx/tree/main/python/ommx-python-mip-adapter), [PyPI](https://pypi.org/project/ommx-python-mip-adapter/)) を使用して、そのインスタンスの最適解と最適値を求めるPythonコードは次の通りです："
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from ommx_python_mip_adapter import instance_to_model, model_to_solution\n",
+    "\n",
+    "# ommx.v1.instanceオブジェクトからPython-MIPのModelオブジェクトに変換する\n",
+    "model = instance_to_model(instance)\n",
+    "# Python-MIPのデフォルトソルバー (SCIP) で最適化を実行する\n",
+    "model.optimize()\n",
+    "# Python-MIPによる最適化計算の結果をommx.v1.Solutionオブジェクトとして取得する\n",
+    "solution = model_to_solution(model, instance)\n",
+    "\n",
+    "# 最適値は41である\n",
+    "assert np.isclose(solution.raw.objective, 41.0)\n",
+    "# x0 = x3 = 1, x1 = x2 = x4 = x5 = 0\n",
+    "assert solution.raw.state.entries == {0: 1, 1: 0, 2: 0, 3: 1, 4: 0, 5: 0}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "補足として、 `ommx.v1.Solution` の `decision_variables` プロパティを使うことで、pandas.DataFrame オブジェクトとして最適解を取得・表示できます:"
```

Comment:
```suggestion
    "補足として、 `ommx.v1.Solution` の `decision_variables` プロパティを使うことで、`pandas.DataFrame` オブジェクトとして最適解を取得・表示できます:"
```

Reply:
fix in https://github.com/Jij-Inc/JijModeling-Tutorials/pull/6/commits/cb9142ec00c6ad89abf8381d8fc053b9327c6f15